packages/components/src/Card/tests/Card.test.tsx

```diff
@@ -50,6 +51,82 @@ it("renders a card", () => {
   `);
 });
 
+it("renders a card with button", () => {
```

Comment:
Almost forgot about this, is there a way to avoid the snapshot? I know we do that on other tests but those tests are also 3-4 years old. We've learned our lesson there to not use snapshots. 😁